apis/r/DESCRIPTION

```diff
@@ -7,7 +7,7 @@ Description: Interface for working with 'TileDB'-based Stack of Matrices,
     from and export to in-memory formats used by popular toolchains like
     'Seurat', 'Bioconductor', and even 'AnnData' using the companion Python
     package.
-Version: 0.1.3.9006
+Version: 0.1.12
```

Comment:
Can we make it 0.1.3.9007 and then integrate the first 'proof of light' C++ bindings from R?

Reply:
@eddelbuettel thanks!

This is on the `main-old` branch not `main` -- completely independent of all `main` development involving C++ etc.

Reply:
`main` is (for now) effectively a `dev` branch

Reply:
The motivation here is to sync the python/R api versions. You could bump the version to 0.1.13.9000 in your PR.

Reply:
See also
**https://github.com/single-cell-data/TileDB-SOMA/wiki/Developer-resources**

Reply:
>  This is on the main-old branch 

Gee. Likely not the last time I fell into _that_ pit.  Never mind then.